presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorAccessControl.java

```diff
@@ -287,4 +293,14 @@ default void checkCanRevokeTablePrivilege(ConnectorTransactionHandle transaction
     {
         denyRevokeTablePrivilege(privilege.toString(), tableName.toString());
     }
+
+    /**
+     * Check if identity is allowed to select from the specified columns in a table.
```

Comment:
Technically, this should be `columns in a relation` as this applies to both tables and views.